DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CourseSetup/AdminFieldsViewModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup
+{
+    using DigitalLearningSolutions.Data.Models.CustomPrompts;
+    using DigitalLearningSolutions.Web.ViewModels.Common;
+
+    public class AdminFieldsViewModel : DisplayPromptsViewModel
+    {
+        public AdminFieldsViewModel(CourseCustomPrompts courseCustomPrompts) : base(courseCustomPrompts.CourseAdminFields)
+        {
+        }
+    }

```

Comment:
Do we need this + DisplayRegistrationPromptsViewModel or could we use the DisplayPromptsViewModel directly?

Reply:
In theory, no. I decided to make these like this to allow for common properties to be carried over, but while keeping the view models specific to the view. I was expecting that we might get more pages in future that would make use of the DisplayPromptsViewModel, but require additional properties specific to their own models. So I set it up like this as an example of what should be done. Would we prefer to just use the DisplayPromptsViewModel?

Reply:
I agree that doing this when/if we have additional properties makes sense, but no reason to create an extra layer of abstraction now that we don't need to specify anything extra. :)